<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motion Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/res/viewer.ico</normaloff>:/icon/res/viewer.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionChinese"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionNdi"/>
    <addaction name="actionHolo"/>
    <addaction name="action3d"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNdiViewer"/>
   <addaction name="actionHoloViewer"/>
   <addaction name="actionKinectViewer"/>
   <addaction name="actionRegi"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="ndiCommDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>NDI Comm</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="ndiDockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="holoCommDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>HoloLens Comm</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="holoDockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="regiDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Registration</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QToolBar" name="glToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDisplayMode"/>
   <addaction name="actionProjection"/>
   <addaction name="separator"/>
   <addaction name="actionAxo"/>
   <addaction name="actionFrontView"/>
   <addaction name="actionRightView"/>
   <addaction name="actionTopView"/>
  </widget>
  <action name="actionChinese">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/translations/res/chinese.png</normaloff>:/translations/res/chinese.png</iconset>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/translations/res/english.png</normaloff>:/translations/res/english.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/about.ico</normaloff>:/icon/res/about.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNdiViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/ndi.png</normaloff>:/icon/res/ndi.png</iconset>
   </property>
   <property name="text">
    <string>NDI Communication</string>
   </property>
   <property name="toolTip">
    <string>NDI Communication</string>
   </property>
  </action>
  <action name="actionHoloViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/vr.ico</normaloff>:/icon/res/vr.ico</iconset>
   </property>
   <property name="text">
    <string>HoloLens Communication</string>
   </property>
   <property name="toolTip">
    <string>HoloLens Communication</string>
   </property>
  </action>
  <action name="actionKinectViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/kinect.ico</normaloff>:/icon/res/kinect.ico</iconset>
   </property>
   <property name="text">
    <string>KinectViewer</string>
   </property>
   <property name="toolTip">
    <string>Kinect Viewer</string>
   </property>
  </action>
  <action name="actionRegi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/regi.svg</normaloff>:/icon/res/regi.svg</iconset>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </action>
  <action name="actionDisplayMode">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/fill.png</normaloff>:/icon/res/fill.png</iconset>
   </property>
   <property name="text">
    <string>DisplayMode</string>
   </property>
  </action>
  <action name="actionNdi">
   <property name="text">
    <string>NDI Spectra</string>
   </property>
  </action>
  <action name="actionHolo">
   <property name="text">
    <string>HoloLens</string>
   </property>
  </action>
  <action name="action3d">
   <property name="text">
    <string>3D Presentation</string>
   </property>
  </action>
  <action name="actionFrontView">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/fview.png</normaloff>:/icon/res/fview.png</iconset>
   </property>
   <property name="text">
    <string>FrontView</string>
   </property>
   <property name="toolTip">
    <string>Front View</string>
   </property>
  </action>
  <action name="actionRightView">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/rview.png</normaloff>:/icon/res/rview.png</iconset>
   </property>
   <property name="text">
    <string>RightView</string>
   </property>
   <property name="toolTip">
    <string>Right View</string>
   </property>
  </action>
  <action name="actionTopView">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/tview.png</normaloff>:/icon/res/tview.png</iconset>
   </property>
   <property name="text">
    <string>TopView</string>
   </property>
   <property name="toolTip">
    <string>Top View</string>
   </property>
  </action>
  <action name="actionAxo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/cube_wire.svg</normaloff>:/icon/res/cube_wire.svg</iconset>
   </property>
   <property name="text">
    <string>AxonometricView</string>
   </property>
   <property name="toolTip">
    <string>Axonometric View</string>
   </property>
  </action>
  <action name="actionProjection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/res/persp.png</normaloff>:/icon/res/persp.png</iconset>
   </property>
   <property name="text">
    <string>Projection</string>
   </property>
   <property name="toolTip">
    <string>Perspective or Orthogonal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
